css/form.css

```diff
@@ -0,0 +1,177 @@
+main {
+  width: 100%;
+  display: flex;
```

Comment:
* The index.html and form.html shares the same navigation header so remember to import the "Robot" font style? Put it into a common CSS file so all HTML could reference?